tree/ntuple/test/CustomStructLinkDef.h

```diff
@@ -154,4 +154,10 @@
 
 #pragma link C++ class RelativelyLargeStruct + ;
 
+#pragma link C++ class v1::Vector3D+;
+#pragma link C++ class v1::ExampleMC+;
+#pragma link C++ class v2::ExampleMC+;
+#pragma read sourceClass = "v1::ExampleMC" source = "v1::Vector3D fSpin" targetClass = \
```

Comment:
```suggestion
#pragma read sourceClass = "v1::ExampleMC" source = "v1::Vector3D fSpin" version="[1-]" targetClass = \
```
If I remember correctly rootcling complains without the 'version' tag.